decred_wallet/Features/Security/PinInputView.swift

```diff
@@ -37,17 +38,20 @@ class PinInputView: UIView {
     }
     
     override func draw(_ frame: CGRect) {
-        self.drawCells(in: frame)
-    }
-    
-    func drawCells(in frame: CGRect) {
+        
         // clear current views
         self.layer.sublayers?.removeAll()
+        
         if self.subviews.count > 0 {
-            self.subviews[0].removeFromSuperview()
+            self.subviews.forEach{ $0.removeFromSuperview() }
         }
         
-        if pin.count > PinInputView.maxNumberOfPinCircles {
+        self.drawCells(in: frame)
+    }
+    
+    func drawCells(in frame: CGRect) {
+        
```

Comment:
Don't see a need for this empty line.

Reply:
Fixed in commit e4174d51aed804a470b8d42f08229756f8993915